ee_tests/README.md

```diff
@@ -35,8 +35,16 @@ npm run image:build
 npm run image:start
 ```
 
+### Running the EE Tests Locally against static html file stored on filesystem ###
+During the test development, it is sometimes convenient to run somd subset of test against locally stored static html page. This is especially useful during debuggin of css selectors. 
 
+#### Implement tests ####
+Add implementation to `src/local/local.ts` test
 
+#### Run the tests ####
+```
+npm start local
+```
```

Comment:
Maybe add a section saying that `local.ts` is not final destination of your test... you should move it elsewhere.